gradle.properties

```diff
@@ -1,7 +1,14 @@
-org.gradle.daemon=true
-org.gradle.jvmargs=-Xms512M -Xmx1G -XX:MaxMetaspaceSize=1G
+org.gradle.daemon=false
+org.gradle.jvmargs=-Xms512M -Xmx1G -Dfile.encoding=UTF-8 -Dconsole.encoding=UTF-8
 org.gradle.configureondemand=false
-gdxVersion=1.13.1
+ktxVersion=1.13.1-rc1
+kotlinVersion=2.0.21
 aiVersion=1.8.2
 ashleyVersion=1.7.4
-ktxVersion=1.13.1-rc1
+artemisOdbVersion=2.3.0
```

Comment:
I guess you had some extra libs enabled because of these versions? We don't need it for this project.